src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -373,7 +373,7 @@ public void setCompressionParametersJson(String options)
     @VisibleForTesting
     public ColumnFamilyStore(Keyspace keyspace,
                              String columnFamilyName,
-                             int generation,
+                             java.util.function.Supplier<? extends SSTableUniqueIdentifier> fileIndexGenerator,
```

Comment:
This is not related with the changes, but maybe we can remove the unused `offline` argument here.

Reply:
I'd not do that in this PR because it may turn out that the lack of usage of `offline` is a bug here